infrastructure/k8s/letsencrypt-clusterissuer.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: cert-manager.io/v1alpha2
+kind: ClusterIssuer
+metadata:
+  namespace: cover-service
+  name: letsencrypt
+spec:
+  acme:
+    server: https://acme-v02.api.letsencrypt.org/directory # production
+    #server: https://acme-staging-v02.api.letsencrypt.org/directory # staging
```

Comment:
Remove?

Reply:
Nope, it's there to ensure that we can remeber the stg url for the stg cluster